src/main/java/com/dumbdogdiner/stickyapi/bungeecord/util/Sound.java

```diff
@@ -0,0 +1,1008 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bungeecord.util;
+
+import lombok.Getter;
+
+/**
+ * An Enum of Sounds the server is able to send to players.
+ * <p>
+ * These values are subject to change on every new major Minecraft release
+ */
+public enum Sound {
```

Comment:
Please deprecate anything related to sound on BungeeCord as it's difficult to maintain and will likely be removed in a future release.

Reply:
Why did we add it in the first place?